src/shared/components/DropdownMenu.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useRef, useEffect } from 'react';
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+import { useMediaQuery } from 'react-responsive';
+
+const years = ['2022/2023', '2021/2022', '2020/2021', '2019/2020', '2018/2019'];
```

Comment:
Can you try making this a dynamic variable? i.e. take the list of years from the parent component.

Reply:
Good idea!

Reply:
Was about to do that, but since we're going to use this dropdown for execs and past events, it's better to have a static constant variable. This way, it makes things more consistent.